meshtastic/tests/test_mesh_interface.py

```
@@ -964,3 +966,264 @@ def test_timeago_fuzz(seconds: int) -> None:
+    """Test that queue operations are thread-safe."""
+    with MeshInterface(noProto=True) as iface:
+        iface.queue = OrderedDict()
+        errors = []
+
+        def add_to_queue(start_id: int) -> None:
+            try:
+                for i in range(50):
+                    packet_id = start_id * 100 + i
+                    packet = mesh_pb2.ToRadio()
+                    iface.queue[packet_id] = packet
+            except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)